File: warehouse-server/schemas/stock.js
'use strict'

var Mongoose = require('mongoose'),
    schema = new Mongoose.Schema({
        isbn: {
            type: String,
            required: true,
            unique: true
        },
        _id_order: {
            type: String
        },
        quantity: {
            type: Number,
            required: true
        }
    });

var item = Mongoose.model('Stock', schema);

module.exports = schema;

File: win-clients/store-client/Management/Stock.cs
namespace store_client.Management
{
    public class Stock
    {

        public string _id_order { get; set; }
        public string isbn { get; set; }
        public int quantity { get; set; }
        public bool accepted { get; set; }

        public Stock()
        {

        }
    }
}
